### PR TITLE
[EZP-28461] Pagination design consistency

### PR DESCRIPTION
**Pull request by inakijv**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-28461](https://jira.ez.no/browse/EZP-28461)
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Most important aspects:
- Design goals:
    - Customer request -> When a Notification message shows up it blocks pagination buttons.
    - Harmonize Pagination and Load more features.

- Design specifications:
    - Pagination text that specifies viewed items and items in total should have the same font-size as in Load more feature;
    - Pagination buttons should have the same font-size as regular buttons in the application (= 1rem)
    - Add extra space below pagination feature, in order to avoid that notification bars block this feature.

bookmarks
content_tab_drafts
content_tab_policies
content_tab_roles
content_tab_urls
content_types_list
languages
link http___caniuse com
link manager
role_list
roles_assignments
roles_policies
search
section_assigned
sections_list
trash

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/_pagination.scss [hunk 1]
```diff
@@ -2,6 +2,18 @@
     font-size: $ez-pagination-font-size;
 }
 
+.ez-pagination-text {
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_pagination.scss]: This is `.scss` file so, you should write the selectors the following way:
```scss
.ez-pagination {
    &-btn {
        font-size: .75rem;
    }
}
```

File: src/bundle/Resources/public/scss/_pagination.scss [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+    font-size: .75rem;
+}
+
+.ez-pagination-btn {
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_pagination.scss]: The next thing is, you're not following BEM naming convention here.
The selectors in this file should look like:
```scss
.ez-pagination__btn {}
.ez-pagination__spacing {}
.ez-pagination__text {}
```

**Comment by inakijv** [comment]
Updated.

**Comment by inakijv** [comment]
Ping @sunpietro

**Pull request merged**